great_expectations/validator/metric_configuration.py

```diff
@@ -11,20 +13,23 @@ def __init__(
         metric_name: str,
         metric_domain_kwargs: dict,
         metric_value_kwargs: Optional[dict] = None,
-        metric_dependencies: Optional[dict] = None,
     ) -> None:
-        self._metric_name = metric_name
+        self._metric_name: str = metric_name
+
         if not isinstance(metric_domain_kwargs, IDDict):
             metric_domain_kwargs = IDDict(metric_domain_kwargs)
-        self._metric_domain_kwargs = metric_domain_kwargs
+
+        self._metric_domain_kwargs: IDDict = metric_domain_kwargs
+
         if not isinstance(metric_value_kwargs, IDDict):
             if metric_value_kwargs is None:
                 metric_value_kwargs = {}
+
             metric_value_kwargs = IDDict(metric_value_kwargs)
-        self._metric_value_kwargs = metric_value_kwargs
-        if metric_dependencies is None:
-            metric_dependencies = {}
-        self._metric_dependencies = metric_dependencies
+
+        self._metric_value_kwargs: IDDict = metric_value_kwargs
+
+        self._metric_dependencies: Optional[Dict[str, MetricConfiguration]] = None
```

Comment:
```suggestion
        self.metric_dependencies:Dict[str, MetricConfiguration] = {}
```

Can we get rid of the `metric_dependencies` property?

Reply:
@Kilo59 If we do it the way you are suggesting, then I think yes.  Do you think this is worth it (i.e., setting it to `dict()` instead of `None`)?  Thanks.